Makefile

```diff
@@ -4,13 +4,14 @@ VERSION := $(shell ./version.sh)
 endif
 REPOSITORY := gravitational.io
 NAME := stolon-app
-OPS_URL ?= https://opscenter.localhost.localdomain:33009
+OPS_URL ?=
 TELE ?= $(shell which tele)
 GRAVITY ?= $(shell which gravity)
```

Comment:
We can probably cut the path munging in the .drone.yml/Jenkinsfile if we update these to reference `./bin/...`

Reply:
I can't do that. For example, I am using Nix and I created shell.nix to download binaries and put them in nix store from which I can point them later if needed.